game/theater/controlpoint.py

```diff
@@ -644,6 +675,20 @@ def process_turn(self, game: Game) -> None:
                             u.position.x = u.position.x + delta.x
                             u.position.y = u.position.y + delta.y
 
+    def allocated_aircrat(self, transfers: PendingTransfers) -> AircraftAllocations:
+        on_order = {}
+        for unit_bought, count in self.pending_unit_deliveries.units.items():
+            if issubclass(unit_bought, FlyingType):
+                on_order[unit_bought] = count
+
+        transferring: dict[Type[FlyingType], int] = defaultdict(int)
+        for transfer in transfers:
+            if transfer.destination == self:
+                for unit_type, count in transfer.units.items():
+                    transferring[unit_type] += count
```

Comment:
This is counting ground units. See `expected_aircraft_next_turn` for an example of counting aircraft transfers. You'll want to rework `ControlPoint.aircraft_transferring` to return a dict instead of just a total.

Reply:
The thing is, that expected_airctaft_next_turn is also used in the QAircraftRecruitmentMenu, where the count is needed. It would not be a problem to change it, or to calculate the numbers from the dic there, but thats the reason I made a new method that worked just like allocated_ground_units.

Reply:
No, I mean these lines (L684-L688) are counting transferring _ground_ units, not air units. `unit_type` here is not a `FlyingType`. `transfer` is a `TransferOrder`, and `TransferOrder.units` is a `Dict[Type[VehicleType], int]`, so you're counting tanks here, not planes. I have no idea why the type checker couldn't catch this, because everything in this chain is explicitly typed :(

But yes, as for fixing it, replace `expected_airctaft_next_turn` with this, with the fixed counting for aircraft transfers.